autoload/deoplete/handlers.vim

```diff
@@ -51,6 +51,8 @@ function! s:completion_begin(event) abort "{{{
   " Save the previous position
   let g:deoplete#_context.position = context.position
 
+  call deoplete#mappings#_set_completeopt()
```

Comment:
Pushed a fix for this in https://github.com/blueyed/deoplete.nvim/commit/69d645dbcfb86af6bf2fa0d4f3c2294f1a09e449.